### PR TITLE
$= phasor/ata, #^ radioInterferometryc99

### PR DESCRIPTION
**Pull request by radonnachie**
* Use astrom for boresight calc_ha_dec too
* Use new `calc_position_delays` from radiointerferometryc99

### CODE REVIEW

File: src/modules/phasor/ata.cc [hunk 1]
```diff
@@ -83,16 +83,23 @@ ATA<OT>::ATA(const typename Generic<OT>::Config& config,
...
 
-    //  Convert Boresight RA & Declination to Hour Angle & Declination.
-    calc_ha_dec_rad(
-        this->config.boresightCoordinate.RA,
-        this->config.boresightCoordinate.DEC,
+    // Convert source RA & Declination to Hour Angle.
+    calc_independent_astrom(
         this->config.arrayReferencePosition.LON,
-        this->config.arrayReferencePosition.LAT,
+        this->config.arrayReferencePosition.LAT,
```
Comment (luigifcruz) [inline src/modules/phasor/ata.cc:92]: Can you remove the blank space at the end?

File: src/modules/phasor/ata.cc [hunk 2]
```diff
@@ -136,32 +126,27 @@ Result ATA<OT>::preprocess(const cudaStream_t& stream) {
...
             source_ha_dec.HA,
             source_ha_dec.DEC,
-            this->config.arrayReferencePosition.LON);
+            this->config.arrayReferencePosition.LON,
+            this->output.delays.data() + (b * this->config.numberOfAntennas)
+        );
 
-        //  Calculate delay for off-center source and subtract 
-        //  from boresight (TPi = Ti - ((WPi - WPr) / C)).
+        //  subtract boresight (TPi = ((WPi - WPr) / C) - Ti).
```
Comment (luigifcruz) [inline src/modules/phasor/ata.cc:147]: Uppercase. "Subtract [...]".

**Review by luigifcruz — COMMENTED** [review]
Thanks, @radonnachie! Can you bump the version of this branch to `0.6.1`?

**Comment by radonnachie** [comment]
Sorted! 👍

**Pull request merged**